### ISSUE TITLE
Circolari e Eventi non appaiono in home

### ISSUE THREAD

**Issue by multiformeingegno** [italia/design-scuole-wordpress-theme#606]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Descrivi il problema

Circolari ed Eventi non appaiono in home nonostante la Home sia configurata per visualizzare entrambi. Circolari sono settate per essere viste da tutti

### Includi link o screenshot

<img width="810" alt="circolari-home" src="https://github.com/italia/design-scuole-wordpress-theme/assets/4039005/2e7d6336-105b-4f74-8b41-f201f0a2811d">

### Su quale sisitema operativo stai riscontrando il problema?

Windows

### Su quale browser stai riscontrando il problema?

Qualsiasi

### Che versione del tema stai utilizzando?

2.8.1

**Comment by sviluppoRobyone** [comment]
Buon pomeriggio, appare almeno il titolo del blocco tipo "Prossimi eventi" o "Circolari" o neanche quello?

**Comment by multifello** [comment]
> Buon pomeriggio, appare almeno il titolo del blocco tipo "Prossimi eventi" o "Circolari" o neanche quello?

Ciao Roby, no… il sito è liceoalighieri.edu.it

**Comment by marcorubin** [comment]
Hai almeno un articolo presente (anche solo nel pannello admin)? Tante volte è legato alla mancanza di un articolo

**Comment by multiformeingegno** [comment]
> Hai almeno un articolo presente (anche solo nel pannello admin)? Tante volte è legato alla mancanza di un articolo

No. Ho aggiunto ora un articolo selezionando come tipologia "Notizie", ma comunque non e' apparso nulla in Home.

**Comment by Simsimus** [comment]
Hai verificato in Configurazione > Novità se la tipologia Notizie è presente fra quelle da mostrare in home?

**Comment by multiformeingegno** [comment]
> Hai verificato in Configurazione > Novità se la tipologia Notizie è presente fra quelle da mostrare in home?

Ah! Era quello il problema! Ora funziona. Grazie dell'aiuto!
Forse andrebbe spostata questa impostazione nella sezione Home?

**Comment by sviluppoRobyone** [comment]
Essendo quell'impostazione relativa anche alla panoramica delle Novità, non so se sia il caso. Conta che nell'opzione in cui decidere se mostrare i contenuti automatici, c'è anche il link
